deploy/cloud-init.yaml

```diff
@@ -11,13 +11,6 @@ users:
       - gh:enolfc
       - gh:gwarf
 
-apt:
```

Comment:
This was added to have a recent ansible (xenial one that we were getting was too old), so I think this should stay

Reply:
But current image is 22.04, right?
https://github.com/EGI-Federation/fedcloud-catchall-operations/blob/58c38e2db0507677b6e8f1538921105dc984bb99/deploy/NCG-INGRID-PT.tfvars#L7

Which is coming with a recent ansible version?
https://packages.ubuntu.com/jammy/ansible

Reply:
ok, let's merge